Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/UiMatcher.kt

```diff
@@ -249,13 +257,22 @@ object UiMatcher {
                 if (expectedDrawable == null || targetDrawable == null) {
                     return false
                 }
-                val expectedBitmap: Bitmap = (expectedDrawable as BitmapDrawable).bitmap
+                if (expectedDrawable::class != targetDrawable::class) {
+                    return false
+                }
+
                 if (targetDrawable is BitmapDrawable) {
-                    val bitmap: Bitmap = targetDrawable.bitmap
-                    return bitmap.sameAs(expectedBitmap)
+                    val targetBitmap: Bitmap = targetDrawable.bitmap
+                    val expectedBitmap = (expectedDrawable as BitmapDrawable).bitmap
+                    return targetBitmap.sameAs(expectedBitmap)
+                } else if (targetDrawable is VectorDrawable) {
+                    val targetBitmap: Bitmap = vectorToBitmap(expectedDrawable as VectorDrawable)
+                    val expectedBitmap = vectorToBitmap(targetDrawable)
```

Comment:
Looks like target and expected are mixed up here.

Reply:
Has been fixed!